tests/test_annotated.py

```diff
@@ -328,7 +328,7 @@ class Model(BaseModel):
         Model(x=2)
     # insert_assert(exc_info.value.errors(include_url=False))
     assert exc_info.value.errors(include_url=False) == [
-        {'type': 'less_than', 'loc': ('x',), 'msg': 'Input should be less than 4', 'input': 2, 'ctx': {'lt': '4'}}
+        {'type': 'less_than', 'loc': ('x',), 'msg': 'Input should be less than 4', 'input': 2, 'ctx': {'lt': 4}}
```

Comment:
We used to stringify errors, now no longer doing it which makes the divergent code paths better match.